Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [lib]
-crate-type = ["cdylib"]
+path = "src/lib.rs"
```

Comment:
this can be omitted as `src/lib.rs` is the default path for `lib` type crates anyway.